buildconfig/pygame-stubs/image.pyi

```diff
@@ -13,5 +13,5 @@ def fromstring(
     flipped: Optional[bool] = False,
 ) -> Surface: ...
 def frombuffer(
-    string: str, size: Union[List[int], Tuple[int, int]], format: str
+    bytes: bytes, size: Union[List[int], Tuple[int, int]], format: str
```

Comment:
I don't think there is a buffer protocol type yet. https://github.com/python/typing/issues/593
I asked on Typing sig about it, maybe someone will have some good idea.

Maybe our own buffer type could start with something like this:
```
from pygame.bufferproxy import BufferProxy
Union[BufferProxy, bytes, bytearray, memoryview]
```
This doesn't work with array.array, or numpy.ndarray, but would be a good start.

Reply:
I added this to the `image.pyi` file as a `_BufferStyle` type similar to the `RectStyle` type we already have.